Sources/KeyboardShortcuts/Recorder.swift

```diff
@@ -114,7 +114,7 @@ extension KeyboardShortcuts.Recorder<Text> {
 	- Parameter onChange: Callback which will be called when the keyboard shortcut is changed/removed by the user. This can be useful when you need more control. For example, when migrating from a different keyboard shortcut solution and you need to store the keyboard shortcut somewhere yourself instead of relying on the built-in storage. However, it's strongly recommended to just rely on the built-in storage when possible.
 	*/
 	public init(
-		_ title: String,
+		_ title: LocalizedStringKey,
```

Comment:
If `@_disfavoredOverload` can't be used in this package, this line can be changed to something like: `key title: LocalizedStringKey` to remove the conflict between this init and the init that accepts StringProtocol.